### PR TITLE
Feature/Lists

### PR DESCRIPTION
**Pull request by SalonSikdar**

### CODE REVIEW

File: pages/components/List/list.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  flex-direction: column;
+  padding-left: 0;
+  margin-bottom: 0;
+  border-radius: 0.25rem;
+}
+
+.list-group-item {
+  position: relative;
+  display: block;
+  padding: 0.5rem 1rem;
```
Comment (anasm11) [inline pages/components/List/list.css]: very small values(<1rem) can be changed to px

**Review by anasm11 — APPROVED** [review]
other types of lists like inline list can be added. The code & UI look good.

File: css/main.css [hunk 2]
```diff
@@ -95,3 +95,17 @@ button:hover {
...
   }
 }
+.d-flex {
+  display: -ms-flexbox !important;
+  display: flex !important;
+}
+
+.justify-content-between {
+  -ms-flex-pack: justify !important;
+  justify-content: space-between !important;
```
Comment (srishtigautam21) [inline css/main.css:105]: Don't use !important

**Review by srishtigautam21 — COMMENTED** [review]
don't use !important
Appreciation: The UI looks great

File: pages/components/List/list.css (hunk 1 above)
Comment (SalonSikdar) [reply pages/components/List/list.css]: Ack, done.

**Comment by SalonSikdar** [comment]
> 
This is a nested PR of the previous one, I'll review all the !important tags in a follow-up PR.

**Pull request merged**